src/message.ts

```diff
@@ -132,6 +134,16 @@ export interface MsgRawObj {
    * MsgType == CONF.MSGTYPE_VERIFYMSG
    */
   RecommendInfo?:   RecommendInfo,
+
+  /**
+   * Transpond Message
```

Comment:
Do those attributes all exist in the WebWxApp data structure?

Need to confirm them too.